### PR TITLE
feat: 🔥 add options to random array

### PR DESCRIPTION
**Pull request by tomalaforge**
add option to random array function to give user to possibility to
return an array of element from a given element array

✅ Closes: #152

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: 152

## Does this PR introduce a breaking change?

```
[] Yes
[x] No
```

### CODE REVIEW

File: packages/falso/src/index.ts [hunk 1]
```diff
@@ -1,23 +1,35 @@
 export { randAbbreviation } from './lib/abbreviation';
```
Comment (NetanelBasal) [inline packages/falso/src/index.ts:1]: Did you remove the `rand`  function from here?

File: packages/falso/src/lib/rand-array.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * @example
+ *
+ * randArray([1, 2, 3])
+ *
+ * @example
+ *
+ * randArray([1, 2, 3], { length: 10 })
+ *
+ */
+export function randArray<T, Options extends FakeOptions = never>(
```
Comment (NetanelBasal) [inline packages/falso/src/lib/rand-array.ts:17]: This name sounds like a function that returns a random array, which isn't always the case.

We can change the `rand` function to `_rand` and leave this as `rand`. You're welcome to suggest a different name.

Comment (tomalaforge) [reply packages/falso/src/lib/rand-array.ts]: I'm not a big fan of _function for internal function. 

I named it randElt if it's ok for you

Comment (NetanelBasal) [reply packages/falso/src/lib/rand-array.ts]: What is elt?

Comment (tomalaforge) [reply packages/falso/src/lib/rand-array.ts]: random element

Comment (NetanelBasal) [reply packages/falso/src/lib/rand-array.ts]: Let's call it randElement

**Pull request merged**